### DISCUSSION TITLE
NES Turbo Fire cant make the knee attack happen in Double Dragon 2.

### DISCUSSION THREAD

**Discussion by alexgame8818** [darthcloud/BlueRetro#1060] [Q&A]
Thanks again for your great project, it has saved so many controllers for me.

In Double Dragon 2, by continuously pressing Turbo A and Turbo B, we can easily perform a knee attack using a turbo fire controller.
But I can't make it happen with BlueRetro's Turbo Fire function, even though I tried all the frame settings.
Other games work fine.

Is it not fast enough, or is it not able to make A and B work at the same time? (The knee attack requires pressing A and B in a very short time when the character lands on the ground after jumping.)

Could you please give some advice about Turbo Fire settings？
Thank you:)

**Comment by darthcloud** [comment]
Hard for me to tell TBH. Would need to know what kind of timing frame wise the game expect. OR what kind of pattern the wired controller do.
